src/libraries/System.Text.Json/src/System/Text/Json/Serialization/JsonSerializerOptions.cs

```diff
@@ -652,28 +652,24 @@ internal void InitializeForReflectionSerializer()
             IsInitializedForReflectionSerializer = true;
         }
 
-        private JsonTypeInfo GetJsonTypeInfoFromContextOrCreate(Type type)
+        private JsonTypeInfo? GetJsonTypeInfoFromResolver(Type type)
```

Comment:
Following naming convention from other methods that wrap validation logic around virtual calls, this might be called
```suggestion
        private JsonTypeInfo? GetTypeInfoInternal(Type type)
```